linux_os/guide/system/software/sudo/sudo_no_others_execute_permission/rule.yml

```diff
@@ -0,0 +1,31 @@
+documentation_complete: true
+
+prodtype: rhel7,rhel8
+
+title: 'Ensure only owner and groupowners can excute sudo'
+
+description: |-
+    Remove the execute permission bit of <tt>/etc/bin/sudo</tt> for the other users.
+    {{{ describe_file_permissions(file="/usr/bin/sudo", perms="4110") }}}
+
+rationale: |-
+    Restricting the set of users able to execute commands as privileged user reduces the attack surface.
+
+severity: medium
+
+identifiers:
+    cce@rhel7: CCE-83571-0
+    cce@rhel8: CCE-83574-4
+
+references:
+  anssi: BP28(R57)
+
+ocil_clause: '{{{ ocil_clause_file_permissions(file="/usr/bin/sudo", perms="---s--x---") }}}'
+
+ocil: '{{{ ocil_file_permissions(file="/usr/bin/sudo", perms="---s--x---") }}}'
+
+template:
+    name: file_permissions
+    vars:
+        filepath: /usr/bin/sudo
+        filemode: '4110'
```

Comment:
Could it conflict with `rpm_verify_permissions` rule?

Reply:
Hmm, yeah, it surely will.

Reply:
> Could it conflict with `rpm_verify_permissions` rule?

Same thing with the other rule changing group ownership. The ultimate problem is that sudo is a setuid program. Wish that would change for the better, but alas, we are where we are.

Reply:
Well, goal of this PR is not to solve the permission conflicts with another rule.

One way to mitigate this is to force evaluation of `rpm_verify_permission` before all other permission changing rules.
This way, the end result is the expected (permissions are reset to default and any deviation is done by the rules in the profile).

The Profile evaluation wont' ever converge, though. Every time a scan is done, the Profile will reset the file permissions according to RPM and then change file permissions according to the profile.
On a second though, I guess there is no way around this, unless we tracked all file permissions in the system, or maybe used AIDE to detect these kinds of changes...

Reply:
In 8048bf9 I force `rpm_verification` group to be first in the Benchmark.
This way the file permissions are reset to default state before the profile hardens them.

Is there any other rule that resets configuration to system default?